test/fixtures/dialects/snowflake/create_external_table.sql

```diff
@@ -9,7 +9,7 @@ create or replace external table "_p08"
     pattern='.*[.]parquet.*';
 
 CREATE EXTERNAL TABLE EXTERNAL_TABLES.TRIPS(
-  tripduration integer as try_cast(VALUE:c1::varchar as integer),
+  tripduration integer as try_cast(VALUE:c1::varchar as integer) not null,
```

Comment:
Let's put `null` on some other column here to test both cases

Reply:
Done